programmers-coding-test/soongu/implement/게임개발.js

```diff
@@ -0,0 +1,57 @@
+`use strict`
+
+const rl = require("readline-sync");
+let [ n, m ] = rl.question("공간의 크기를 입력해주세요.\n>> ").split(' ').map((val) => parseInt(val));
+const d = Array.from(Array(n), () => Array(m).fill(0))
+
+let [ x, y , direction ] = rl.question("현재 캐릭터의 x좌표, y좌표, 방향을 입력해주세요.\n>> ").split(' ').map((val) => parseInt(val));;
+d[x][y] = 1;
+
+array = [];
+for (let i = 0; i < n; i++) {
+  array.push(rl.question("육지 바다 구분을 해주세요.\n>> ").split(' ').map((val) => parseInt(val)));
+}
+
+let dx = [ -1, 0, 1, 0 ]; 
+let dy = [ 0, 1, 0, -1 ]; 
+
+function turn_left() {
+  direction -= 1;
+  if (direction === -1) {
+    direction = 3;
+  }
```

Comment:
코드가 다 똑같으니까 더 귀찮은거 같아요...ㅜㅜ
```suggestion
  direction = direction - 1 === -1 ? 3 : direction--;
```